ras_common_utils/ras_config/ras_config.py

```diff
@@ -62,54 +70,82 @@ def items(self):
         return self.service.items()
 
     def dependency(self, k):
+
         try:
             return DependencyProxy(self._dependencies[k], k)
         except KeyError:
             raise RasDependencyError("Dependency with name '{}' not found.".format(k))
 
     def dependencies(self):
+
         return {k: DependencyProxy(self._dependencies[k], k) for k in self._dependencies.keys()}.items()
 
     def features(self):
         return self._features
 
+    def is_cloud_foundry(self):
```

Comment:
I don't think we need to flag this, as the CloudFoundryConfig class encapsulates the detail that we're running on CF, while the 'make' factory function takes on the responsibility of deciding if CloudFoundryConfig is the correct class to instantiate. The consuming code shouldn't need to be aware if it's running on CF.